app/scripts/app.js

```diff
@@ -17,7 +23,8 @@ blocChat.config(['$stateProvider', '$locationProvider', function($stateProvider,
 
 blocChat.factory("chatRooms", [
     "$firebaseArray",
-    function($firebaseArray) {
+    "CurrentRoom",
+    function($firebaseArray, CurrentRoom) {
         var roomsUrl = 'https://stewart-bloc-chat.firebaseio.com/rooms';
         var roomsRef = new Firebase(roomsUrl);
         var roomsObject = $firebaseArray(roomsRef);
```

Comment:
The root of the roomUrl's are the same.  Duplication could be removed there.  I'm not sure why there is a hard-coded value of "citizens" in there, but that seems suspect as well.

That's pretty much all I see that could use improving.  Great job.